app/models/pacbio/well_factory.rb

```diff
@@ -112,94 +102,24 @@ def save
         return false unless valid?
 
         well.save
-        return false unless libraries.save && pools.save
+        return false unless pools.save
 
         true
```

Comment:
Here `well.save` returns false if save fails (unlikely because we just validated, but still possible) and yet we'll ignore that and head on to persist the pool.

This could just become `well.save && pool.save`. That way we'd only save the pool if the well saved, and would return false if either failed. (And true if bot succeeded)

Reply:
Things not quite behaving as expected here. These may not be rails models...

Reply:
Hmm, pools is a factory object... makes sense